modules/upgrade/partials/incompat-changes.adoc

```diff
@@ -1,6 +1,8 @@
 === Review incompatible changes
 
-* *Breaking change in Redpanda 25.3*: Schema Registry no longer allows specifying a schema ID and version when registering a schema in read-write mode. You must use import mode to register a schema with a specific ID and version. See xref:manage:schema-reg/schema-reg-api.adoc#set-schema-registry-mode[Use the Schema Registry API] for more information.
+* *Breaking changes in Redpanda 25.3*:
+** Redpanda v25.3 introduces breaking schema changes for Iceberg topics. Before upgrading, review xref:manage:iceberg/iceberg-schema-changes-and-migration-guide.adoc[] and follow the required migration steps to avoid data loss or query errors for your Iceberg topics.
```

Comment:
"Redpanda v25.3 introduces breaking schema changes for Iceberg Topics. If you are already using Iceberg Topics and want to maintain the data in the corresponding Iceberg tables, before upgrading your cluster, review xref:manage:iceberg/iceberg-schema-changes-and-migration-guide.adoc[] and follow the required migration steps to avoid sending new topic records to a Dead Letter Queue table."